lego-webapp/components/CommandPalette/commands.tsx

```diff
@@ -0,0 +1,203 @@
+          icon: <Icon iconNode={<Banana />} size={15} />,
+        },
+        {
+          id: 'joblistings',
+          label: 'Jobbannonser',
+          action: () => navigate('/joblistings'),
+          icon: <Icon iconNode={<Newspaper />} size={15} />,
+        },
+        {
+          id: 'companies',
+          label: 'Bedrifter',
+          action: () => navigate('/companies'),
+          icon: <Icon iconNode={<Briefcase />} size={15} />,
+        },
+        {
+          id: 'articles',
+          label: 'Artikler',
+          action: () => navigate('/articles'),
+          icon: <Icon iconNode={<BookOpen />} size={15} />,
+        },
+        {
+          id: 'gallery',
+          label: 'Album',
+          action: () => navigate('/photos'),
+          icon: <Icon iconNode={<BookImage />} size={15} />,
+        },
+        {
+          id: 'quotes',
+          label: 'Overhørt',
+          action: () => navigate('/quotes'),
+          icon: <Icon iconNode={<Quote />} size={15} />,
+        },
+        {
+          id: 'trophies',
+          label: 'Trofeer',
+          action: () => navigate('/trophies'),
+          icon: <Icon iconNode={<Trophy />} size={15} />,
+        },
+        {
+          id: 'the fund',
+          label: 'Fondet',
+          action: () => window.open('https://fondet.abakus.no/', '_blank'),
+          icon: <Icon iconNode={<Landmark />} size={15} />,
+        },
+        {
+          id: 'developerblog',
+          label: 'Utviklerbloggen',
+          action: () => window.open('https://webkom.dev/', '_blank'),
+          icon: <Icon iconNode={<MessageSquareQuote />} size={15} />,
+        },
+        {
+          id: 'settings',
+          label: 'Innstillinger',
+          action: () => navigate('/users/me/settings/profile'),
+          icon: <Icon iconNode={<Settings />} size={15} />,
+        },
+      ],
+    },
+    {
+      name: 'Kommandoer',
+      items: [
+        {
+          id: 'createMeetingNotice',
+          label: 'Lag møteinnkalling',
+          action: () => navigate('/meetings/new'),
+          icon: <Icon iconNode={<Terminal />} size={15} />,
+        },
+        {
+          id: 'createReceipt',
+          label: 'Lag kvittering',
+          action: () => window.open('https://kvittering.abakus.no/', '_blank'),
+          icon: <Icon iconNode={<Terminal />} size={15} />,
+        },
+        {
+          id: 'createQuote',
+          label: 'Lag sitat',
+          action: () => navigate('/quotes/new'),
+          icon: <Icon iconNode={<Terminal />} size={15} />,
+        },
+        {
+          id: 'createAlbum',
+          label: 'Lag album',
+          action: () => navigate('/photos/new'),
+          icon: <Icon iconNode={<Terminal />} size={15} />,
+        },
+        {
+          id: 'createLendingobject',
+          label: 'Lag utlånsobjekt',
+          action: () => navigate('/lending/new'),
+          icon: <Icon iconNode={<Terminal />} size={15} />,
```

Comment:
Since not all users can create lending objects, we should hide this for those users.